packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte

```diff
@@ -67,5 +81,10 @@
             @apply grid;
             grid-template-columns: 2fr 1fr 1fr 1fr;
         }
+
+        info-section {
+            @apply px-5 py-1.5 gap-2;
+            background-color: rgba(212, 161, 0, 0.1);
```

Comment:
You can now use the opacity of the collor like this:  
```suggestion
            @apply bg-yellow/10;
```

Reply:
ah yes, that wasnt available yet at that point. ty

Reply:
also warning instead of yellow 🙏